kolibri/core/content/api.py

```diff
@@ -1246,12 +1246,13 @@ def filter_available(self, queryset, name, value):
         queryset = queryset.annotate(
             available=Subquery(
                 models.ContentNode.objects.filter(
-                    available=True,
                     id=OuterRef("contentnode_id"),
                 ).values_list("available", flat=True)[:1]
             )
         )
 
+        return queryset.filter(available=value)
+
```

Comment:
What's the difference between returning `available=value` vs. filtering by `available=True`?

Reply:
If `value` is `False` then it would only return not available ones (although, technically it would need to be `False` or `None`) so it wouldn't work quite as expected for the negative case.